dist/focus-ring.js

```diff
@@ -278,42 +237,49 @@ document.addEventListener('DOMContentLoaded', function() {
    * @param {Element} el
    */
   function removeFocusRingClass(el) {
-    if (!el.hasAttribute('data-focus-ring-added'))
```

Comment:
I see your commit to add these back in, but for some reason the diff is showing these are still removed.